validators/migrate-vali/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Migrate Validator"
+description: >
+  How to migrate an Autonity validator node to a new instance or to a new IP/Port address
+---
+
+::: {.callout-note title="Prerequisites" collapse="false"}
+- Ensure that the host machine meets the [minimum requirements](/node-operators/install-aut/#requirements)
+:::
+
+
+## Migrating validator node to a new environment
+
+To migrate a node to a new instance follow the steps described in the node operator's guide [Migrate Autonity](/node-operators/migrate-aut/).
+
+::: {.callout-warning title="Important" collapse="false"}
+If your node is in an active state, you must [pause the validator node](/validators/pause-vali/) **before migration**. [Reactivate the validator](/validators/pause-vali/) **after migration**.
+
+Be sure to fully decommission the original node installation!
+:::
+
+
+## Migrating validator node to a new IP/Port address
+
+This guide describes how to migrate a validator node to an environment with a new IP/Port network address by updating the validator's registered enode URL.
+
+::: {.callout-warning title="Important" collapse="false"}
+Only the network connection information - IP and/or port - of a registered validator enode can be updated. You **cannot** change the validator's address (i.e. the `PUBKEY` part of the [validator enode URL](/concepts/validator/#validator-enode-url)).
+:::
+
+When performing the update the validator **must not** not be an active member of the consensus committee and should be in a [`paused`](/concepts/validator/#validator-pausing) state.
+
+
+### Step 1: Pause the validator
+
+Pause the validator as described in the guide [Pause as a validator](/validators/pause-vali/#pause-as-a-validator).
+
+If your validator is an active member of the consensus committed, wait for the epoch to end before proceeding further.
```

Comment:
We can add a important note here. Validator must make sure before updating the enode. that is no longer part of the consensus committee

Reply:
Done in commit ed9c3d7d66e1b8ec343e21f854e8c8ed20f19bbc.